x-pack/dockerlogbeat/Dockerfile

```diff
@@ -1,18 +1,4 @@
-ARG versionString
-FROM golang:${versionString} as builder
-
-WORKDIR /go/src/github.com/elastic/beats/x-pack/dockerlogbeat
-COPY . ../..
-
-ENV GOPATH=/go
-ARG GOOS=linux
-ARG GOARCH=amd64
-ARG GOARM=
-
-RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o dockerlogbeat
-
-
-FROM alpine:3.7 as final
+FROM alpine:3.7
 RUN apk --no-cache add ca-certificates
-RUN mkdir /contmount
-COPY --from=builder /go/src/github.com/elastic/beats/x-pack/dockerlogbeat/dockerlogbeat /usr/bin/dockerlogbeat
+COPY build/dockerlogbeat /usr/bin/
+
```

Comment:
Can you please add the same labels used in other container images. See https://github.com/elastic/beats/blob/75ddc22031b5cb4e1a881216a9fd452040f16230/dev-tools/packaging/templates/docker/Dockerfile.tmpl#L7-L18

I'm assuming these labels pass-through into what ever the plugin builder produces.

Reply:
I mean, technically that container is build artifact. The final product produced by `mage package` is a tarball that gets turned into a runc container. There won't be any labels in the final product, if that's what you're asking.